Marlin/ConfigurationStore.cpp

```diff
@@ -128,6 +136,11 @@ void Config_StoreSettings()  {
   EEPROM_WRITE_VAR(i, max_e_jerk);
   EEPROM_WRITE_VAR(i, add_homing);
 
+  #if defined(MESH_BED_LEVELING)
+    EEPROM_WRITE_VAR(i, mbl.active);
+    EEPROM_WRITE_VAR(i, mbl.z_values);
+  #endif  // MESH_BED_LEVELING
```

Comment:
Ah, ok I see now. Though I see some other options do not do that, but most do. Maybe one should add a length check when reading to read defaults if storage length has changed?